src/UI/Core/TextField.cpp

```diff
@@ -139,13 +139,13 @@ void TextField::onTextInput(const std::string& _s)
 
 	if (mMaxCharacters > 0 && text().length() == mMaxCharacters) { return; }
 
-	int prvLen = text().length();
+	std::size_t prvLen = text().length();
```

Comment:
This change appears to be included in #173. It may be more appropriate in this PR.

Reply:
Done